src/pages/home/Search.tsx

```diff
@@ -54,7 +56,22 @@ function Search() {
   const isAuthorized = useSelector(
     (state: IRootState) => state.authorize.isAuthorized,
   )
+  const isemail = useSelector((state: IRootState) => state.email.email)
 
+  useEffect(() => {
+    UserDetailsService.getUserDetails(isemail)
+      .then((response) => {
+        if (response.data.email === isemail) {
+          dispatch(fetchUserDetails(response.data))
+        } else {
+          alert('user not found')
+        }
+      })
+      .catch((error) => {
+        console.log(error)
+        navigate('/errorpage')
```

Comment:
This error page currently used for invalid routes. we can't navigate to this page.